src/plots/ternary/index.js

```diff
@@ -75,3 +75,11 @@ exports.clean = function(newFullData, newFullLayout, oldFullData, oldFullLayout)
         }
     }
 };
+
+exports.updateFx = function(gd) {
+    var fullLayout = gd._fullLayout;
+    
```

Comment:
Please run `npm run lint -- --fix` and commit the changes.

```suggestion
```

Reply:
![image](https://github.com/user-attachments/assets/dfafaf5b-9ccf-4c75-bb22-468e1fcca707)

I hope I'm doing it correctly.